### ISSUE TITLE
Closing issues in PRs/commits

### ISSUE THREAD

**Issue by bobbysebolao** [ReginaldJbeili/feline-fun-mr-project#25]
Well done for referencing issue numbers in your commit messages, this will come in handy soon when you’re managing more issues!

Did you know you can use keywords in your PR/commit messages to close issues automatically when the PR is merged to master? From the Github docs: 

_The following keywords, followed by an issue number, will close the issue:_
* _close_
* _closes_
* _closed_
* _fix_
* _fixes_
* _fixed_
* _resolve_
* _resolves_
* _resolved_

_For example, to close an issue numbered 123, you could use the phrase "Closes #123" or "Closes: #123" in your pull request description or commit message. Once the branch is merged into the default branch, the issue will close._

https://help.github.com/en/github/managing-your-work-on-github/closing-issues-using-keywords

**Comment by ReginaldJbeili** [comment]
noted thanks!
